components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -85,10 +86,15 @@ def _configure(self, parser):
             "Output a config template for web server\n"
             "  nginx: Nginx system configuration for inclusion\n"
             "  nginx-development: Standalone user-run Nginx server\n"
+            "  nginx-wsgi: Standalone user-run Nginx server\n"
+            "  nginx-wsgi-development: Standalone user-run Nginx server\n"
             "  apache: Apache 2.2 with mod_fastcgi\n"
-            "  apache-fcgi: Apache 2.4+ with mod_proxy_fcgi\n")
+            "  apache-fcgi: Apache 2.4+ with mod_proxy_fcgi\n"
+            "  apache-wsgi: Apache 2.4+ with mod_wsgi\n")
```

Comment:
What's our plan for supporting apache versions in future? `mod_wsgi` works on Centos 6 with Apache 2.2, but if we're dropping support for 2.2 then this can stay as 2.4+.